### PR TITLE
OFFICAL BAN OVERHAUL PR

### PR DESCRIPTION
**Pull request by ghost**
+Fixed stray apostrophe left in ban redirect.
+Deleted .htaccess based banning, switched to SQL based banning.
+Fixed a bug where deleting parent posts did not remove the children from the management panel or database
+Check if the user is banned everytime they post
+Reordered SQL config values info for no reason
+Every ban is perma-posting ban for now, unless someone goes into the
SQL table and manually deletes the entry

COMING SOON:
-Ability to remove bans
-Ban redirect page with reason why, IP banned from
-Automated ban removal after time served is up
- code is hacky as SHIT
  Hopefully I'll have something else neat done tonight ¯_(ツ)_/¯

### CODE REVIEW

File: config.php [hunk 1]
```diff
@@ -81,7 +81,7 @@
 define(PHP_SELF, 'imgboard.php');						//Name of main script file
 define(PHP_SELF2, 'index.html');						//Name of main htm file
 define(PHP_EXT, '.html');								//Extension used for board pages after first
-define(JS_PATH, 'jquery'); 								//relative path from imgboard.php of the jquery folder without a trailing slash
+define(JS_PATH, 'plugins/jquery'); 								//relative path from imgboard.php of the jquery folder without a trailing slash
```
Comment (RePod) [inline config.php:88]: Wouldn't it be better to define this after PLUG_PATH then inherit it?

Comment (ghost) [inline config.php:88]: Whatever's easiest and does the least damage, I didn't want to mess with the suite

**Comment by ghost** [comment]
I'll leave all this here to be merged in case I get hit by a bus or something and disappear. Otherwise, don't merge this yet until the ban system is complete. I'll keep adding to it as I go along.

**Comment by ghost** [comment]
I also recommend deleting kusaba and monotone .css and adding them to the CSS fun pack dlc, they're kinda old and I don't know how popular they'd be.

aight, all done

**Comment by RePod** [comment]
Not really translation friendly at the moment, but I guess that's the biggest problem with having something support translations.

As for the CSS, there's no reason to remove them. If and when a more scalable CSS system is put into place the issue will resolve itself and give the people running it the choice.

**Pull request merged**